src/hooks/dialogmote/useInitialValuesReferat.ts

```diff
@@ -42,8 +43,9 @@ export const useInitialValuesReferat = (
             funksjon,
           })
         ),
-        standardtekster: referatTexts.standardTekster.filter((standardtekst) =>
-          latestReferat.document.some(({ key }) => key === standardtekst.key)
+        standardtekster: getReferatTexts(Malform.BOKMAL).standardTekster.filter(
```

Comment:
Må vi hente for valgt målform her også? (selv om det ikke spiller noen rolle når man bare bruker `key` under)